org-formation/050-costs/categories.yaml

```diff
@@ -28,151 +28,7 @@ Resources:
-            Accounts:
-              - "216152803258"
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "101600"
-          - Value: '112501 Mobile Toolbox Project Core'
-            Accounts:
-              - "634761300905"
-              - "611413694531"
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "112501"
-          - Value: '314900 iAtlas'
-            Accounts:
-              - "386990716034"
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "314900"
-          - Value: '120100 HTAN DFCI'
-            Accounts:
-              - "888810830951"
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "120100"
-          - Value: '101500 NIA AMP AD CC'
-            Accounts:
-              - "607346494281"
-              - "681175625864"
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "101500"
-          - Value: '101700 NIA AMP AD Supp Multi Scale'
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "101700"
-          - Value: '106000 UMass PECDCC'
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "106000"
-          - Value: '122500 MGH RECOVER'
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "122500"
-          - Value: '312000 Genie AACR'
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "312000"
-          - Value: '312100 Genie AACR BPC'
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "312100"
-          - Value: '315200 Wellcome Leap 1kd'
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "315200"
-          - Value: '40033 Genentech imCore'
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "40033"
-          - Value: '000000 NO PROGRAM'
-            TagNames:
-              - CostCenterOther
-              - CostCenter
-            TagEndsWith:
-              - "000000"
-        InheritedValues:
-          TagOrder:
-            - CostCenterOther
-            - CostCenter
-          RulePosition: Last
+      'Fn::Transform':
+        Name: 'AWS::Include'
+        Parameters:
+          Location: 's3://sagebase-rules-microservice-outputbucket-17vzcnk2xnq0/cost-categories/program-code-rules.yaml'
```

Comment:
why not include owner-email-rules.yaml for `OwnerEmailCostCategory` resource as well?

Reply:
because it's generating invalid rules. we can create rules that inherit email addresses from tags, but trying to create account-based rules where the category name is an email address fails a regex constraint (regex details are in it-2369). i have an open aws support ticket but i'm not hopeful that the category name regex will be expanded to match the tag value regex any time soon :(

Reply:
so i thought i saw some code in the lambda to supply OwnerEmailCostCategory?  what's the plan for that?  are we going to keep that code in the lambda and just not use it?

Reply:
yeah I would like to use it eventually, if AWS ever relaxes the category regex constraint to match the regex constraint for tags. so my plan is to leave the functionality in place with hopes of eventually being able to use it

Reply:
I'm not a fan of unused code dangling around.  if you must keep it in then could you either comment it out completely or at least provide a comment explaining that it's not used for whatever reason?

Reply:
is it enough to comment out the lambda in the template? https://github.com/Sage-Bionetworks-IT/lambda-finops-cost-rules/pull/8

or do i need to remove `cost_emails/owner_emails.py` from the lambda entirely?

Reply:
It's up to you.  It should either be completely removed or completely commented out with an explanation as to why it's not used.